adafruit_button.py

```diff
@@ -197,10 +196,10 @@ def label(self, newtext):
         dims = self._label.bounding_box
         if dims[2] >= self.width or dims[3] >= self.height:
             raise RuntimeError("Button not large enough for label")
-        self._label.x = self.x + (self.width - dims[2]) // 2
-        self._label.y = self.y + self.height // 2
+        self._label.x = 0 + (self.width - dims[2]) // 2
+        self._label.y = 0 + self.height // 2
```

Comment:
```suggestion
        self._label.x = (self.width - dims[2]) // 2
        self._label.y = self.height // 2
```

I think these zeroes will be confusing going forwards.